### ISSUE TITLE
Build failed in step 'Installing final gcc compiler'

### ISSUE THREAD

**Issue by yc2986** [crosstool-ng/crosstool-ng#1003]
# Missing header in "install final gcc compiler" step
- **crosstool version**: crosstool-ng-1.23.0
- **Host OS**: macOS High Sierra 10.13.6
- **Config file**: [config.zip](https://github.com/crosstool-ng/crosstool-ng/files/2216326/config.zip)
- **build.log**: [build_log.zip](https://github.com/crosstool-ng/crosstool-ng/files/2216327/build_log.zip)

Specific error message in build.log:  

> [ERROR]    /Volumes/xtool-build-env/.build/armv8-rpi3-linux-gnueabihf/build/build-cc-gcc-final/armv8-rpi3-linux-gnueabihf/libstdc++-v3/include/bits/stl_algobase.h:66:42: fatal error: bits/stl_iterator_base_funcs.h: No such file or directory

Appreciated if anyone can provide some instruction. As I was following an [old guide](https://www.jaredwolff.com/blog/cross-compiling-on-mac-osx-for-raspberry-pi/) to setup cross compile for RPi 3. I was using the brew version of crosstool-ng. Thanks in advance!

**Comment by yc2986** [comment]
**Update**:  
After rerun ```ct-ng build``` several times I got different error messages regarding missing different headers (like stdexcpet and the above one). I think in my 3rd re-try it mysteriously succeeded by itself. Could anyone provide some analysis on if this is the correct behavior? Thanks.

**Comment by 0x6e** [comment]
I also received this error, but the build was actually failing because the image I was installing into was too small. Once I fixed the space available to install into I still got the error below, but the build continued.
```[INFO ]  Installing final gcc compiler
[EXTRA]    Configuring final gcc compiler
[EXTRA]    Building final gcc compiler
[ERROR]    clang: error: unsupported option '-print-multi-os-directory'
[ERROR]    clang: error: no input files
[EXTRA]    Installing final gcc compiler
[EXTRA]    Housekeeping for final gcc compiler
[EXTRA]       '' --> lib (gcc)   lib (os)
[INFO ]  Installing final gcc compiler: done in 299.00s (at 22:57)```

**Comment by stilor** [comment]
Is it on APFS filesystem? If so, it is probably a duplicate of #906

**Comment by yc2986** [comment]
@stilor yes it is on APFS filesystem. I was able to successfully compile by expanding the volume size. Thanks
